apps/api/src/controllers/v1/game-modes.controller.ts

```
@@ -132,4 +148,11 @@ export const deleteGameModeController = async (c: Context) => {
     throw new ApiError(errorMessage, 404);
   }
   return c.json(convertBigIntToString(gameMode) as object);
+  return c.json(
+    {
+      success: true,
+      message: "Game mode deleted successfully",
+    },
+    201
+  );
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)